### PR TITLE
Added entries to sys.configurations view

### PR DESCRIPTION
**Pull request by faviansamatha**
### Description
Previously, there were missing values in the sys.configurations which blocked the Table - Create To feature in SSMS. This commit fixes that issue by adding values to sys.babelfish_configurations, the underlying table for sys.configurations view.

Task: BABELFISH-296
Signed-off-by: Favian (Ian) Samatha <ians@bitquilltech.com>

### Check List
- [X] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: test/JDBC/logfile [hunk 1]
```diff
@@ -0,0 +1,156 @@
+2022-06-10 21:32:28.502 UTC [14887] LOG:  starting PostgreSQL 14.3 on x86_64-pc-linux-gnu, compiled by gcc (GCC) 7.3.1 20180712 (Red Hat 7.3.1-14), 64-bit
```
Comment (Deepesh125) [inline test/JDBC/logfile]: Seems like this file is included by mistake.

**Review by Deepesh125 — CHANGES_REQUESTED** [review]
Add test cases where object is referenced inside a view, procedure and function to tests dependency for upgrade. We have developed framework to test upgrade which should be used to add test cases in prepare/verify and cleanup format

File: test/JDBC/upgrade/13_6/preparation/sys-configurations-vu-prepare.sql [hunk 2]
```diff
@@ -0,0 +1,47 @@
+CREATE DATABASE db_sys_configurations;
```
Comment (surendravishnoi) [inline test/JDBC/upgrade/13_6/preparation/sys-configurations-vu-prepare.sql]: Ideally, we should create test objects with test name and that way no need to use test specific database

**Review by Deepesh125 — APPROVED** [review]
.

**Review by hsuamz — APPROVED** [review]

**Review by rossd-bitquill — APPROVED** [review]

**Pull request merged**